app/uk/gov/hmrc/gform/config/FrontendAppConfig.scala

```diff
@@ -39,7 +39,8 @@ case class FrontendAppConfig(
   routeToSwitchLanguage: String => play.api.mvc.Call,
   contactFormServiceIdentifier: String,
   optimizelyUrl: Option[String],
-  trackingConsentSnippet: hmrcTrackingConsentSnippet
+  trackingConsentSnippet: hmrcTrackingConsentSnippet,
+  emailAuthDefaultEmailIds: Option[Seq[String]]
```

Comment:
`Option[Seq[String]]` is really weird type. Why not to make it `Option[NonEmptyList[CIString]]` ?